website/blog/2023-06-14-install-ferretdb-mongodb-alternative-ubuntu.mdx

```diff
@@ -0,0 +1,321 @@
+---
+slug: install-ferretdb-mongodb-alternative-ubuntu
+title: 'How to Install FerretDB on Ubuntu'
+author: Klinsmann Öteyo
+description: >
+  This is a guide on how to install FerretDB - the MongoDB alternative - on Ubuntu.
+image: /img/blog/install-ferretdb-ubuntu.jpg
+keywords: [FerretDB, mongodb alternative]
+---
+
+<head>
+  <link rel="canonical" href="https://tutornix.com/how-to-install-ferretdb-mongodb-alternative-on-ubuntu-22-0420-04/" />
+</head>
+
+![How to install FerretDB on Ubuntu](/img/blog/install-ferretdb-ubuntu.jpg)
+
+_This is a duplicate post from [Tutornix](https://tutornix.com/how-to-install-ferretdb-mongodb-alternative-on-ubuntu-22-0420-04/) about installing FerretDB on Ubuntu; we are grateful to the author for the permission to repost this on our blog._
+
+<!--truncate-->
+
+Welcome to this guide on how to install [FerretDB](https://www.ferretdb.io/) - MongoDB Alternative - on Ubuntu.
+But before we dive into the nub of this matter, we need to know what FerretDB is!
+
+Databases are majorly categorised as Relational and Non-relational.
+Today, our key focus is on non-relational databases(also known as NoSQL).
+These databases do not observe the traditional tabular relational database model of rows and columns with pre-defined schemas, but instead use a variety of data models, such as key-value, document, column-family, and graph models, to store and manage data.
+
+They are often used to store and manage unstructured or semi-structured data, such as text, images, videos, social media posts, and sensor data, that may not fit well into traditional relational databases.
+They are known for better scalability, performance, and availability than traditional relational databases, particularly for web-scale applications with high volumes of data and traffic.
+The most popular NoSQL databases are [MongoDB](https://www.mongodb.com/), [Redis](https://redis.io/), [Couchbase](https://www.couchbase.com/), [Cassandra](https://cassandra.apache.org/_/index.html), [Neo4j](https://neo4j.com/) etc.
+
+FerretDB is a free and open-source alternative to MongoDB, built on [Postgres](https://www.postgresql.org/).
+It translates the MongoDB wire protocol queries to SQL with the help of PostgreSQL as the database engine.
```

Comment:
Ok, I don't know how much we want to edit the original blog post, but that sentence is incorrect: PostgreSQL does not help FerretDB to translate MongoDB wire protocol queries to SQL.